src/inputs/labeled-inputs.scss

```diff
@@ -4,45 +4,36 @@
 @import '../icon/index';
 
 @mixin iui-input-container {
+  @include iui-reset;
   display: flex;
   flex-direction: column;
-  flex-wrap: nowrap;
   position: relative;
-  padding: 0;
-  margin: 0;
   text-align: left;
-  font-size: $iui-font-size;
   font-weight: $iui-font-weight-normal;
   cursor: default;
 
   // Pointer cursor used if this class is applied to a label element
   @at-root {
     label#{&} {
       cursor: pointer;
+    }
+  }
 
-      &.iui-disabled {
-        cursor: not-allowed;
-      }
+  &.iui-disabled {
+    &,
+    label {
+      cursor: not-allowed;
     }
   }
 
   > .iui-label {
-    padding: 0;
     margin-bottom: $iui-component-offset;
-    border: none;
-    display: block;
-    width: fit-content;
-    font-size: $iui-font-size;
     font-weight: $iui-font-weight-semibold;
-    line-height: $iui-line-height;
-    text-align: left;
-    @include themed {
-      color: t(iui-text-color);
-    }
 
     &.iui-required {
       &::after {
         content: ' *';
+        margin-right: 0;
```

Comment:
Why this was needed?

Reply:
This counteracts the extra margin set on non-required label

Reply:
Oh actually this is leftover, I think. Will remove

Reply:
still not removed

Reply:
? 
![image](https://user-images.githubusercontent.com/9084735/131501844-abc7d114-69b7-48d9-a3c6-1c672d00cc2f.png)

Reply:
amm.. ok. I was schecking, and it was still there.. Now i see its not.. weird.